common/app/experiments/Experiments.scala

```diff
@@ -58,3 +59,12 @@ object DarkModeWeb
       sellByDate = LocalDate.of(2025, 4, 30),
       participationGroup = Perc0D,
     )
+
+object DCRFootballLive
```

Comment:
So we'll have one experiment for each page? (live/results/fixtures?) That makes sense and we can roll them out separately 👍

Reply:
Good question, we can actually have a generic experiment instead of football/live 🤔 Maybe it's better I call it `DCRFootball`?